go.mod

```diff
@@ -28,7 +28,7 @@ require (
 	github.com/ory/dockertest/v3 v3.10.0
 	github.com/osmosis-labs/go-mutesting v0.0.0-20221208041716-b43bcd97b3b3
 	github.com/osmosis-labs/osmosis/osmomath v0.0.9-0.20240222171503-685566578734
-	github.com/osmosis-labs/osmosis/osmoutils v0.0.9-0.20240222004208-b602d1901059
+	github.com/osmosis-labs/osmosis/osmoutils v0.0.9-0.20240304224617-9042f6d1e24f
```

Comment:
Can we also change other osmoutil instances from different `go.mod` in our repo to this commit? (really should not matter, but kind of feels weird having different commits of osmoutils being imported)